### ISSUE TITLE
version 1.2.0 escapeTransformDescription not found

### ISSUE THREAD

**Issue by developer-devPHP** [zfcampus/zf-apigility-documentation#47]
**Last version (1.2.0)** have this kind of error. 
When I want just open the specific documentation, I got an error

**URL:** 
apigility/documentation/```myapi-v1```

function usage is in file:
```
/vendor/zfcampus/zf-apigility-documentation/view/zf-apigility-documentation/operation.phtml
 line 41

 <td><?php echo $this->escapeTransformDescription($field->getDescription()) ?></td>
```
Can't find  ```escapeTransformDescription()``` function

ERROR output

**File:**
```
/var/www/vendor/zendframework/zend-servicemanager/src/ServiceManager.php                    :555
```
**Message:**
```
Zend\View\HelperPluginManager::get was unable to fetch or create an instance for escapeTransformDescription
```
**Stack trace:**
```
#0 /var/www/vendor/zendframework/zend-servicemanager/src/AbstractPluginManager.php(161): Zend\ServiceManager\ServiceManager->get('escapeTransform...', true)
#1 /var/www/vendor/zendframework/zend-view/src/Renderer/PhpRenderer.php(373): Zend\ServiceManager\AbstractPluginManager->get('escapeTransform...', NULL)
#2 /var/www/vendor/zendframework/zend-view/src/Renderer/PhpRenderer.php(391): Zend\View\Renderer\PhpRenderer->plugin('escapeTransform...')
#3 /var/www/vendor/zfcampus/zf-apigility-documentation/view/zf-apigility-documentation/operation.phtml(41): Zend\View\Renderer\PhpRenderer->__call('escapeTransform...', Array)
#4 /var/www/vendor/zfcampus/zf-apigility-documentation/view/zf-apigility-documentation/operation.phtml(41): Zend\View\Renderer\PhpRenderer->escapeTransformDescription('')
#5 /var/www/vendor/zendframework/zend-view/src/Renderer/PhpRenderer.php(502): include('/var/www/vendor...')
#6 /var/www/vendor/zfcampus/zf-apigility-documentation/view/zf-apigility-documentation/service.phtml(25): Zend\View\Renderer\PhpRenderer->render('zf-apigility-do...', Array)
#7 /var/www/vendor/zendframework/zend-view/src/Renderer/PhpRenderer.php(502): include('/var/www/vendor...')
#8 /var/www/vendor/zfcampus/zf-apigility-documentation/view/zf-apigility-documentation/api.phtml(15): Zend\View\Renderer\PhpRenderer->render('zf-apigility-do...', Array)
#9 /var/www/vendor/zendframework/zend-view/src/Renderer/PhpRenderer.php(502): include('/var/www/vendor...')
#10 /var/www/vendor/zfcampus/zf-apigility-documentation/view/zf-apigility-documentation/show.phtml(6): Zend\View\Renderer\PhpRenderer->render('zf-apigility-do...', Array)
#11 /var/www/vendor/zendframework/zend-view/src/Renderer/PhpRenderer.php(502): include('/var/www/vendor...')
#12 /var/www/vendor/zendframework/zend-view/src/View.php(207): Zend\View\Renderer\PhpRenderer->render(Object(Zend\View\Model\ViewModel))
#13 /var/www/vendor/zendframework/zend-view/src/View.php(236): Zend\View\View->render(Object(Zend\View\Model\ViewModel))
#14 /var/www/vendor/zendframework/zend-view/src/View.php(200): Zend\View\View->renderChildren(Object(Zend\View\Model\ViewModel))
#15 /var/www/vendor/zendframework/zend-mvc/src/View/Http/DefaultRenderingStrategy.php(105): Zend\View\View->render(Object(Zend\View\Model\ViewModel))
#16 [internal function]: Zend\Mvc\View\Http\DefaultRenderingStrategy->render(Object(Zend\Mvc\MvcEvent))
#17 /var/www/vendor/zendframework/zend-eventmanager/src/EventManager.php(490): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#18 /var/www/vendor/zendframework/zend-eventmanager/src/EventManager.php(251): Zend\EventManager\EventManager->triggerListeners('render', Object(Zend\Mvc\MvcEvent))
#19 /var/www/vendor/zendframework/zend-mvc/src/Application.php(384): Zend\EventManager\EventManager->triggerEvent(Object(Zend\Mvc\MvcEvent))
#20 /var/www/vendor/zendframework/zend-mvc/src/Application.php(356): Zend\Mvc\Application->completeRequest(Object(Zend\Mvc\MvcEvent))
#21 /var/www/public/index.php(129): Zend\Mvc\Application->run()
#22 {main}
```

**Comment by developer-devPHP** [comment]
I think it should be ```agTransformDescription()```

**Comment by weierophinney** [comment]
Fixed with #50.
